components/eventing-controller/controllers/subscription/beb/reconciler_test.go

```diff
@@ -896,11 +1004,7 @@ var _ = Describe("Subscription Reconciliation Tests", func() {
 
 			Context("Delete BEB Subscription", func() {
 				By("Deleting its entry in BEB mock internal cache", func() {
-					for k := range beb.Subscriptions {
-						if strings.Contains(k, nameMapper.MapSubscriptionName(kymaSubscription)) {
-							delete(beb.Subscriptions, k)
-						}
-					}
+					beb.Subscriptions.DeleteSubscriptionsByName(nameMapper.MapSubscriptionName(kymaSubscription))
```

Comment:
I think renaming `beb` to `bebMock` would improve the readability.

Reply:
I considered this too but in this scope there is another variable dubbed `bebMock`. I will rename 
`bebMock` to `mock` and
`beb` to `bebMock`.
What do you think?